src/components/Lesson.js

```diff
@@ -12,11 +12,24 @@ const Lesson = ({ route }) => {
   const enExample = grammar.examples.find(
     (example) => example.language === "en"
   );
+
+  const navigateBtn = (position) => {
```

Comment:
since grammar is a route params, you don't need to pass in position here. You can just access `grammar.position` directly in this function.